src/classes.py

```diff
@@ -0,0 +1,25 @@
+class Product:
+    def __init__(self, name, description, price, quantity):
```

Comment:
Не проставлены тайпхинты
```suggestion
    def __init__(self, name: str, description: str, price: float, quantity: int):
```
Для категории нужно сделать то же самое